docsource/modules150-160.rst

```diff
@@ -32,7 +32,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | account_qr_code_sepa                            |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| account_sale_timesheet                    |                      |                                                 |
+| |del| account_sale_timesheet                    |Done                  |Merged into sale_project                         |
```

Comment:
```suggestion
| |del| account_sale_timesheet                    |                      |Merged into sale_project                         |
```
@NgoHoangHung  It should be left blank

Reply:
ok, I edited again

Reply:
No , it should be `Done` not blank, because you need to declare it at `merged` section in `apriory.py` file